.github/workflows/build.yml

```diff
@@ -0,0 +1,79 @@
+name: Build
+
+on: push
+
+jobs:
+  patch-src:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: GitHub Environment Variables Action
+        uses: FranzDiebold/github-env-vars-action@v1.2.1
+
+      - name: Set up Debian Docker container for changelog patching
+        # '--ignore-branch' for testing on feature branch
+        run: |
+          docker create --name chglog --volume ${{ github.workspace }}:${{ github.workspace }} --workdir ${{ github.workspace }}/ --tty pitop/gbp-dch-gen:latest sleep inf
```

Comment:
What goes in `pitop/gbp-dch-gen:latest`?

Reply:
That's the docker image, which is generated from this: https://github.com/pi-top/gbp-dch-gen-docker/blob/master/Dockerfile. You can see that it adds a couple of dev packages on top of a Docker image for `buster-slim`. You can change this to `buster-slim` if you like, as you don't make use of the added dev packages!

Reply:
Our custom Docker image is hosted here: https://hub.docker.com/repository/docker/pitop/gbp-dch-gen

Reply:
Yes, I'll give a try to the plain buster image. It is great that Ubuntu 18 is based on it, since I don't support anything older than that (too many OSs installed already). Thanks for the explanation!